src/libretro.h

```diff
@@ -0,0 +1,4405 @@
+/* Copyright (C) 2010-2020 The RetroArch team
```

Comment:
Did you make any changes to this file for the SkyEmu port? Or is it just the straight header from the retro arch project?

Reply:
it is a straight copy